x/project/spec/03_messages.md

```diff
@@ -0,0 +1,149 @@
+# Messages
+
+In this section we describe the processing of the bonds messages and the corresponding updates to the state. All created/modified state objects specified by each message are defined within the [state](./02_state.md) section.
```

Comment:
This mentions bonds but we're not dealing with bonds here.

Reply:
In this section, we describe the processing of the project messages.